client/src/views/organizations/OrganizationInformationPage.vue

```diff
@@ -1,24 +1,150 @@
 <!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
 
 <template>
-  <ion-page>
-    <ion-content :fullscreen="true">
+  <ion-page class="organization-page">
```

Comment:
Way too complex. It should be something closer to:
```
<organization-page-small-display
  v-if="orgInfo && userInfo && smallDisplay"
  :org-info="orgInfo"
  :user-info="userInfo"
>
<organization-page-large-display
  v-if="orgInfo && userInfo && largeDisplay"
  :org-info="orgInfo"
  :user-info="userInfo"
/>
```
This way you can have each page in its own component, you avoid a lot of `v-if="smallDisplay"` everywhere, it makes everything way easier to read.
It's fine to put everything in the same component if it's a small difference, but in this case the page is a lot different depending on the display.